src/crypto-api.ts

```diff
@@ -93,6 +93,15 @@ export interface CryptoApi {
      */
     importRoomKeys(keys: IMegolmSessionData[], opts?: ImportRoomKeysOpts): Promise<void>;
 
+    /**
+     * Import a list of room keys restored from backup
+     *
+     * @param keys - a list of session export objects
+     * @param opts - options object
+     * @returns a promise which resolves once the keys have been imported
+     */
+    importBackedUpRoomKeys(keys: IMegolmSessionData[], opts?: ImportRoomKeysOpts): Promise<void>;
+
```

Comment:
any reason this needs to be here rather than in `CryptoBackend`?

Reply:
Only because I didn't know about `CryptoBackend` ;)